website/docs/replicalimits.md

```diff
@@ -135,7 +135,7 @@ spec:
 Usage
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper-library/master/library/general/replicalimits/samples/replicalimits/example_allowed.yaml
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper-library/master/library/general/replicalimits/samples/replicalimits/constraint.yaml
```

Comment:
I think we need the revert this change

Reply:
Thanks @sozercan - It appears there were manually updated in https://github.com/open-policy-agent/gatekeeper-library/commit/1def56b845b5c7a1acf37b9f611c7e7a8835dd68, and are being changed back "make generate generate-website-docs".  I'll need to look into it further.

Reply:
Hi @sozercan - I traced this to an issue with the website generate script #254 that was impacting all the website policy examples.  I just opened PR #255 to resolve.